examples/step-59/doc/results.dox

```diff
@@ -0,0 +1,341 @@
+   <td>11</td>
+   <td>12</td>
+  </tr>
+  <tr>
+   <th>Number of DoFs</th>
+   <td>2097152</td>
+   <td>7077888</td>
+   <td>16777216</td>
+   <td>32768000</td>
+   <td>7077888</td>
+   <td>11239424</td>
+   <td>16777216</td>
+   <td>23887872</td>
+   <td>32768000</td>
+   <td>43614208</td>
+   <td>7077888</td>
+   <td>8998912</td>
+  </tr>
+  <tr>
+   <th>Number of iterations</th>
+   <td>13</td>
+   <td>12</td>
+   <td>12</td>
+   <td>12</td>
+   <td>13</td>
+   <td>13</td>
+   <td>15</td>
+   <td>15</td>
+   <td>17</td>
+   <td>19</td>
+   <td>18</td>
+   <td>18</td>
+  </tr>
+  <tr>
+   <th>Solver time [s]</th>
+   <td>0.713</td>
+   <td>2.150</td>
+   <td>4.638</td>
+   <td>8.803</td>
+   <td>2.041</td>
+   <td>3.295</td>
+   <td>5.723</td>
+   <td>8.306</td>
+   <td>12.75</td>
+   <td>19.25</td>
+   <td>3.530</td>
+   <td>4.814</td>
+  </tr>
+  <tr>
+   <th>MDoFs/s</th>
+   <td>2.94</td>
+   <td>3.29</td>
+   <td>3.62</td>
+   <td>3.72</td>
+   <td>3.47</td>
+   <td>3.41</td>
+   <td>2.93</td>
+   <td>2.88</td>
+   <td>2.57</td>
+   <td>2.27</td>
+   <td>2.01</td>
+   <td>1.87</td>
+  </tr>
+</table>
+
+We clearly see how the efficiency per DoF initially improves until it reaches
+a maximum for the polynomial degree $k=4$. This effect is surprising, not only
+because higher polynomial degrees often yield a vastly better solution, but
+especially also when having matrix-based schemes in mind where the denser
+coupling at higher degree leads to a monotonously decreasing throughput (and a
+drastic one in 3D, with $k=4$ being more than ten times slower than
+$k=1$!). For higher degres, the throughput decreases a bit, which is both due
+to an increase in the number of iterations (going from 12 at $k=2,3,4$ to 19
+at $k=10$) and due to the $\mathcal O(k)$ complexity of operator
+evaluation. Nonetheless, efficiency as the time to solution would be still
+better at higher degrees because they have better convergence rates (at least
+for problems as simple as this one). For $k=12$, we reach roundoff accuracy
+already at 1 million Dofs (solver time less than a second), whereas for $k=8$
+we need 24 million DoFs and 8 seconds. For $k=5$, the error is around
+$10^{-9}$ at 57m DoFs and thus still far away from roundoff, despite taking 16
+seconds.
+
+Note that the above numbers are a bit pessimistic because they
+include the time it takes the Chebyshev smoother to compute an eigenvalue
+estimate, which is around 10 percent of the solver time.
+
+<h3>Evaluation of efficiency of ingredients</h3>
+
+Finally, we take a look at some of the special ingredients presented in this
+tutorial program, namely the FE_DGQHermite basis in particular and the
+specification of MatrixFree::DataAccessOnFaces. In the following table, the
+third row shows the optimized solver above, the fourth row shows the timings
+with only the MatrixFree::DataAccessOnFaces set to `unspecified` rather than
+the optimal `gradients`, and the last one with replacing FE_DGQHermite by the
```

Comment:
The condition numbers are pretty good, see here:
http://dealii.org/developer/doxygen/deal.II/classPolynomials_1_1HermiteLikeInterpolation.html

It's around twice the condition number of the polynomials in FE_DGQ, which is the close to the best you can do given the additional constraints set up by the Hermite conditions as far as I can tell.

For this particular solver, the basis does not have any effect on iteration counts because the block-Jacobi setup cancels the effect of the basis, so the mathematical properties are completely equivalent (except roundoff). However, things would be different if I had chosen a point-Jacobi smoother (like the basic Chebyshev smoother that takes the matrix diagonal). In that case, multigrid takes 2-3 iterations more with `FE_DGQHermite` than it does with `FE_DGQ`. That's something we should add to the discussion, I will write a new paragraph about that in the introduction.